k8s/service_controller/server/nil/storage.go

```diff
@@ -79,6 +79,10 @@ func (s *NilServiceStorage) GetBrokerByService(id string) (*model.ServiceBroker,
 	return nil, nil
 }
 
+func (s *NilServiceStorage) AllServices() ([]*model.Service, error) {
```

Comment:
can we add it later when needed? Just feels odd if its not used.
We still need the rest of the PR though, right?